Package.swift

```diff
@@ -156,3 +158,13 @@ extension Array where Element == PackageDescription.CXXSetting {
     ]
   }
 }
+
+extension Array where Element == PackageDescription.LinkerSetting {
+  /// Settings intended to be applied to every linked target in this package.
+  /// Analogous to project-level build settings in an Xcode project.
+  static var packageSettings: Self {
```

Comment:
This property is named **package** settings but it's not actually applied to all of the targets currently, just one.

However, it seems like actually only the single target which uses curl needs this linkage. So how about instead of creating this new package-wide setting, we only add this linkage to the single target which requires it?

Reply:
Can do!